demo/demo_director.py

```diff
@@ -742,8 +742,30 @@ def listen():
 
   server.register_function(clear_vehicle_targets, 'clear_vehicle_targets')
 
-  server.register_function(mitm_arbitrary_package_attack, 'mitm_arbitrary_package_attack')
-  server.register_function(undo_mitm_arbitrary_package_attack, 'undo_mitm_arbitrary_package_attack')
+  # Attack 1: Arbitrary Package, no keys
+  server.register_function(mitm_arbitrary_package_attack,
+      'mitm_arbitrary_package_attack')
+  server.register_function(undo_mitm_arbitrary_package_attack,
+      'undo_mitm_arbitrary_package_attack')
+
+  # Attack 2: Replay, no keys
+  server.register_function(prepare_replay_attack_nokeys,
```

Comment:
The definition of rollback provided in the design doc is:
* *Figure 7.3.3a: Rollback attack, where attackers cause ECUs to install obsolete software. The vehicle has version 10 of ECU-1 software, but the attackers send version 1 of the software as the latest update, even though version 12 exists on the repository.*

This attack is replaying valid but outdated metadata.